docs/admin/resourcequota/index.md

```diff
@@ -84,45 +85,77 @@ You might want to set a pods quota on a namespace
 to avoid the case where a user creates many small pods and exhausts the cluster's
 supply of Pod IPs.
 
+## Quota Scopes
+
+Each quota can have an associated set of scopes.  A quota will only measure usage for a resource if it matches
+the intersection of enumerated scopes.
+
+When a scope is added to the quota, it limits the number of resources it supports to those that pertain to the scope.
+Resources specified on the quota outside of the allowed set results in a validation error.
+
+| Scope | Description |
+| ----- | ----------- |
+| Terminating | Match pods where `spec.activeDeadlineSeconds >= 0` |
+| NotTerminating | Match pods where `spec.activeDeadlineSeconds is nil` |
+| BestEffort | Match pods that have best effort quality of service. |
+| NotBestEffort | Match pods that do not have best effort quality of service. |
+
+A `BestEffort` scope restricts a quota to limit the following resources:
+* pods
+
+A `Terminating`, `NotTerminating`, and `NotBestEffort` scope restricts a quota to tracking the following resources:
+* cpu
+* limits.cpu
+* limits.memory
+* memory
+* pods
+* requests.cpu
+* requests.memory
+
+## Requests vs Limits
+
+When allocating compute resources, each container may specify a request and a limit value for either CPU or memory.
+The quota can be configured to quota either value.
+
+If the quota has a value specified for `requests.cpu` or `requests.memory`, then it requires that every incoming
+container makes an explicit request for those resources.  If the quota has a value specified for `limits.cpu` or `limits.memory`,
+then it requires that every incoming container specifies an explict limit for those resources.
+
 ## Viewing and Setting Quotas
 
 Kubectl supports creating, updating, and viewing quotas:
 
 ```shell
-$ kubectl namespace myspace
-$ cat <<EOF > quota.json
-{
-  "apiVersion": "v1",
-  "kind": "ResourceQuota",
-  "metadata": {
-    "name": "quota"
-  },
-  "spec": {
-    "hard": {
-      "memory": "1Gi",
-      "cpu": "20",
-      "pods": "10",
-      "services": "5",
-      "replicationcontrollers":"20",
-      "resourcequotas":"1"
-    }
-  }
-}
+$ kubectl create namespace myspace
+$ cat <<EOF > object-counts.yaml
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: object-counts
+spec:
+  hard:
+    configmaps: "10"
+    persistentvolumeclaims: "4"
+    replicationcontrollers: "20"
+    secrets: "10"
+    services: "10"
+    services.loadbalancers: "2"
 EOF
```

Comment:
Add some newlines after here and some of the other lines below so it doesn't all run together?